rpc/core/net.go

```diff
@@ -31,12 +29,8 @@ func NetInfo() (*ctypes.ResultNetInfo, error) {
 
 //-----------------------------------------------------------------------------
 
-// Dial given list of seeds if we have no outbound peers
+// Dial given list of seeds
```

Comment:
This is unsafe.  We should rename it to Unsafe* or something, otherwise we could forget to close this loophole.